components/brave_wallet_ui/components/shared/add-custom-token-form/add-nft-form.tsx

```diff
@@ -131,6 +131,7 @@ export const AddNftForm = (props: Props) => {
         coingeckoId: '',
         logo: '',
         tokenId: tokenId ? new Amount(tokenId).toHex() : '',
+        isCompressed: false,
```

Comment:
Do we need to add a lookup query to determine this value?

Reply:
Shouldn't - the value will be fetched from simplehash by the backend when the user adds the token. We can do similar lookups too in follow up PRs for all data returned by the simplehash endpoint if we want.

Reply:
Can we add a note here mentioning that the backend will handle the `isCompressed` check?